src/controllers/auth.controller.ts

```diff
@@ -7,13 +7,17 @@ import { JWT_SECRET } from '../configs/envConfig'
 
 export const register = async (req: Request, res: Response): Promise<void> => {
   try {
-    const { email, password } = req.body
+    const { email, password, type } = req.body
 
     if (!email || !password) {
       res.status(400).json({ error: 'Email and password are required fields' })
     }
 
-    const { statusCode, message, profile } = await registerUser(email, password)
+    const { statusCode, message, profile } = await registerUser(
+      email,
+      password,
+      type
+    )
```

Comment:
You should not update this to send the type. With this anyone can become an admin. @Shrenik0321

Reply:
Oh right. understood